petab/visualize/data_overview.py

```diff
@@ -12,12 +14,17 @@
 __all__ = ['create_report']
 
 
-def create_report(problem: petab.Problem, model_name: str) -> None:
+def create_report(
+        problem: petab.Problem,
+        model_name: str,
+        outdir: Union[str, Path] = ''
+) -> None:
     """Create an HTML overview data / model overview report
 
     Arguments:
         problem: PEtab problem
         model_name: Name of the model, used for file name for report
+        outdir: Output directory
     """
 
     template_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)),
```

Comment:
```suggestion
output_path = Path(output_path)
template_path = output_path.parent.absolute() / 'templates'
```
But then may need to use `str(template_path)` later